[ADD] l10n_ar_account_withholding: demo data

zaoral opened the pull request:
pre configure taxes, company withholding regimen and partners.

zaoral committed:
[ADD] l10n_ar_account_withholding: demo data

pre configure taxes, company withholding regimen and partners.

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/765).

zaoral committed:
[FIX] l10n_ar_account_withholding: warning log missing title

Before this change when trying to update or install the module we
receive this warning in odoo log. Adding the title attribute resolve
the warning

WARNING db-ret odoo.addons.base.models.ir_ui_view: A button with icon attribute (fa-print ) must have title in its tag, parents, descendants or have text
View error context:
{'file': '/home/odoo/custom/repositories/ingadhoc-odoo-argentina/l10n_ar_account_withholding/views/account_payment_view.xml',
 'line': 5,
 'name': 'account.payment.tree',
 'view': ir.ui.view(1127,),
 'view.model': 'account.payment',
 'view.parent': ir.ui.view(558,),
 'xmlid': 'view_account_payment_tree'}
